### ISSUE TITLE
attempt to perform arithmetic on field `time' (a nil value)

### ISSUE THREAD

**Issue by InfoDav** [shirsig/Aux-Addon#28]
When trying to do a full auction scan. I often, but not always, get this error during the scan (around page 12-20 over 170-210):

Interface\AddOns\Aux-AddOn\history.lua:76: attempt to perform arithmetic on field `time' (a nil value)

If I click Okay, it is followed by this error:

Interface\AddOns\Aux-AddOn\control.lua:30: attempt to call local `k' (a nil value)

I can click Okay at this message but it will always come back, so I have to close the auction window.

I am not versed at all in Lua and AddOn programming, so didn't take a look at the code. So I am totally unsure about what causes this. But since it doesn't always happen, it might be related to a specific auction listing causing it?

**Comment by shirsig** [comment]
Sounds like you updated from an old version that didn't save the timestamps without clearing the database at some point. Try /aux clear history + /reload.

**Comment by InfoDav** [comment]
You were right. Everything is working perfectly fine now.

**Comment by hammerface** [comment]
Hello, I'm having the same issue InfoDav was having.

![ss 2016-12-14 at 06 53 39](https://cloud.githubusercontent.com/assets/8420389/21186669/403d608a-c1ca-11e6-8201-4979b18e8630.jpg)

I have tried the commands you recommended, but maybe "/aux clear history" has been removed in later versions? I also tried "/aux clear item cache", but I don't know if that is a similar function.

I also deleted the aux-addon.lua in my SavedVariables, but that didn't fix it. This problem started happening after I updated my version of aux, so its probably the same issue but I don't know how to fix it.
